.circleci/config.yml

```diff
@@ -191,3 +231,4 @@ workflows:
       - build-nightly-securedrop-client:
           requires:
              - build-nightly-securedrop-proxy
+      - build-nightly-securedrop-export
```

Comment:
flagging: this should be changed before the next nightly build as it will likely fail - this new nightly job should require `build-nightly-securedrop-proxy` and `build-nightly-securedrop-client` so that these jobs run in sequence, not concurrently, as they will run in about the same amount of time and commit directly to master in the LFS repo